suites/reef/rados/tier-3_rados_test-4-node-ecpools.yaml

```diff
@@ -1,6 +1,116 @@
 # Suite is to be used to verify the EC 2+2 config on 4 nodes
 
 tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574887
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                # using 6.1 rc candidate
+                custom_image: "registry-proxy.engineering.redhat.com/rh-osbs/rhceph:6-177"
+                custom_repo: "http://download-node-02.eng.bos.redhat.com/rhel-9/rel-eng/RHCEPH-6/RHCEPH-6.1-RHEL-9-20230602.0/compose/Tools/x86_64/os/"
+                mon-ip: mero006
+                allow-fqdn-hostname: true
+                yes-i-know: true
```

Comment:
@pdhiran what is the use of the `yes-i-know ` entry?

Reply:
This would be used to ignore FQDN vs Shortnames for hosts. We have seen that hostnames that are set post reimage in octo lab are not consistent with the naming.

Without the arg, bootstrap will fail if the hosts have FQDN set.

Reply:
`allow-fqdn-hostname` and `yes-i-know` are not related to each other AFAIK, could you please confirm?
https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/5/html/installation_guide/red-hat-ceph-storage-installation

Reply:
Removed --yes-i-know flag.

Reply:
The flag was added to override any other warnings that might arise during bootstrap. Let's debug and add back the flag if failure arises.